packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/utils/schema.js

```diff
@@ -152,7 +152,7 @@ const createYupSchema = (
 
         const { max, min } = attribute;
 
-        if (attribute.required && !options.isDraft) {
+        if (attribute.required) {
```

Comment:
I think the required check must stay conditionnal.  just the min that needs to be applied even with draft

Reply:
indeed thank you!
I tried to fix it, does it seem okay to you?

Reply:
I think the dynamic zones were already working as intended actually. We can check together if you want